### DISCUSSION TITLE
Where to file an issue regarding mount issues in devcontainers

### DISCUSSION THREAD

**Discussion by kerryhatcher** [devcontainers/community#92] [Q&A] [answered]
I have an problem with using devcontainers on PopOS with Docker Desktop that I'm unsure on which repo to open an issue on. 

[I've created a hello world example that replicates the issue on two separate machines. ](https://github.com/kerryhatcher/devcontainerdemo)

Should this go in a repo in this org or the vscode under microsoft? 

Thanks!

**Comment by eljog** [comment, marked as answer]
Thank you for trying devcontainers.
I assume this may be a PopOS specific issue since I could not replicate it on a Mac.

If you are using vscode `Dev Containers` extension you can file an issue at https://github.com/microsoft/vscode-remote-release/issues
If you are using the devcontainers CLI, please file an issue at https://github.com/devcontainers/cli/issues

**Reply by kerryhatcher** [reply]
Thanks!!
